src/tests/modules/client/xlat.unlang

```diff
@@ -25,8 +25,8 @@ if (&Tmp-String-0 != 'b_type') {
 #
 #  Test non-existent client properties
 #
-&Tmp-String-3 := "%(client:non-existent-attr)"
-&Tmp-String-4 := "%(client:non-existing-attr2)"
+&Tmp-String-3 := "%client(non-existent-attr)"
```

Comment:
these should be quoted like the first parameter to the other `%client()` calls

Reply:
Done.